lighthouse-core/report/v2/report-styles.css

```diff
@@ -224,25 +224,44 @@ body {
   padding-top: 0;
 }
 
-.lighthouse-category > .lighthouse-audit {
+.lighthouse-category > .lighthouse-audit,
+.lighthouse-category > details > .lighthouse-audit {
```

Comment:
should we use the `.lighthouse-category-passed` class instead of generic `details` for all of these?